spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -494,21 +494,6 @@
         verify_all_vacancy_details(vacancy)
       end
 
-      scenario 'enables the user to resolve cross-form errors' do
```

Comment:
Can't we keep this spec?

Reply:
I removed it because it's testing a scenario that wouldn't ever arise, and the feature specs are long enough already